### DISCUSSION TITLE
Can’t install VirtualBox in Elementary 7.1

### DISCUSSION THREAD

**Discussion by abernardo88** [elementary/triage#543] [Q&A]
Hello, I’m trying to install different versions of virtualbox (last one included) and all failed for dependencies issues. Has anyone faced that issue? Any solutions?
Thanks.

**Comment by doobes** [comment]
I've got VB installed on a EOS v7.1, and v8.0 beta.

Did you follow the instructions here:

https://www.virtualbox.org/wiki/Linux_Downloads
